java/src/jmri/util/JmriJFrame.java

```diff
@@ -352,7 +352,7 @@ public String getWindowFrameRef() {
      * By default, Swing components should be created an installed in this
      * method, rather than in the ctor itself.
      *
-     * @throws java.lang.Exception may throw an Exception if an overriding class
+     * @throws Exception may throw an Exception if an overriding class
```

Comment:
If you remove the throws Excpetion here, I think you will get a compilation error for every subclass that throws Exception, which would allow you to narrow down the non-runtime exceptions you want to propagate as should be caught and dealt with to only those that are really thrown

Reply:
Agreed.  That's the source of about 1/3 of the "throws Exception" occurrences, with half of those coming from `jmri.jmrix.AbstractMonFrame`.  Another big group comes from `jmri.util.swing.JmriPanel.java`.  They're on my list to (slowly) work through.